src/payload/orderPayloadBuilder.ts

```diff
@@ -0,0 +1,69 @@
+import OrderPayload from "./orderPayload";
+import { Sender } from "../models/sender";
+import { Recipient } from "../models/recipient";
+
+export default class OrderPayloadBuilder {
+    quotationId: string | undefined;
+
+    sender: Sender | undefined;
+
+    recipients: Recipient[] | undefined;
+
+    fleetOption?: string;
+
+    isPODEnabled?: boolean;
+
+    isRecipientSmsEnabled?: boolean;
+
+    paymentMethod?: string;
```

Comment:
Same as `fleetOnly`, probably better to not mention this in our SDK

Reply:
ok